### PR TITLE
Add new RPC methods debug_getRawBlock, debug_getRawReceipts, debug_getRawHeader, debug_getRawTransaction

### PR DESCRIPTION
**Pull request by Demuirgos**
Fixes Closes Resolves #

_Please choose one of the keywords above to refer to the issue this PR solves followed by the issue number (e.g. Fixes #000). If no issue number, remove the line. Also, remove everything marked optional that is not applicable. Remove this note after reading._

## Changes

-Adds missing debug_getRaw* methods to Debug module

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [x] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

#### Notes on testing

_Optional. Remove if not applicable._

#### Requires documentation update

- [x] Yes
- [ ] No

_If yes, link the PR to the docs update or the issue with the details labeled `docs`. Remove if not applicable._

#### Requires explanation in Release Notes

- [x] Yes
- [ ] No

_If yes, fill in the details here. Remove if not applicable._

## Remarks

_Optional. Remove if not applicable._

### CODE REVIEW

File: scripts/deployment/archive-packages.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@ cd osx-x64 && zip -r -y $GITHUB_WORKSPACE/$PACKAGE_DIR/$PACKAGE_PREFIX-macos-x64
 cd osx-arm64 && zip -r -y $GITHUB_WORKSPACE/$PACKAGE_DIR/$PACKAGE_PREFIX-macos-arm64.zip . && cd ..
 cd ref && zip -r $GITHUB_WORKSPACE/$PACKAGE_DIR/$PACKAGE_PREFIX-ref-assemblies.zip . && cd ..
 
-echo "Archiving completed"
+echo "Archiving completed"
```
Comment (rubo) [inline scripts/deployment/archive-packages.sh:20]: Please revert this change.

File: scripts/deployment/build-runner.sh [hunk 2]
```diff
@@ -31,4 +31,4 @@ done
 mkdir $output_path/ref
 cp ../artifacts/obj/**/$build_config/refint/*.dll $output_path/ref
 
-echo "Build completed"
+echo "Build completed"
```
Comment (rubo) [inline scripts/deployment/build-runner.sh:34]: Please revert this change.

**Review by rubo — CHANGES_REQUESTED** [review]

**Review by rubo — APPROVED** [review]

File: src/Nethermind/Nethermind.Specs/SystemTransactionReleaseSpec.cs [hunk 3]
```diff
@@ -130,5 +130,7 @@ public bool IsEip158IgnoredAccount(Address address)
         public bool IsEip6780Enabled => _spec.IsEip6780Enabled;
         public bool IsEip4788Enabled => _spec.IsEip4788Enabled;
         public Address Eip4788ContractAddress => _spec.Eip4788ContractAddress;
+
+        public bool AuRaSystemCalls => false;
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Specs/SystemTransactionReleaseSpec.cs:134]: Is this related to this PR?

File: src/Nethermind/Nethermind.TxPool/TxPool.cs [hunk 4]
```diff
@@ -159,12 +159,12 @@ public class TxPool : ITxPool, IDisposable
         public IDictionary<Address, Transaction[]> GetPendingTransactionsBySender() =>
             _transactions.GetBucketSnapshot();
 
-        public IDictionary<Address, Transaction[]> GetPendingLightBlobTransactionsBySender() =>
-            _blobTransactions.GetBucketSnapshot();
-
         public Transaction[] GetPendingTransactionsBySender(Address address) =>
             _transactions.GetBucketSnapshot(address);
 
+        public IDictionary<Address, Transaction[]> GetPendingLightBlobTransactionsBySender() =>
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.TxPool/TxPool.cs:165]: why move?

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.TxPool/TxPool.cs (hunk 4 above)
Comment (Demuirgos) [reply src/Nethermind/Nethermind.TxPool/TxPool.cs]: merge conflict and I ended up with two copies of the same method, I removed the first one and left the duplicate by mistake

File: scripts/deployment/build-runner.sh
Comment (MarekM25) [inline scripts/deployment/build-runner.sh:1]: ?

File: scripts/deployment/archive-packages.sh
Comment (MarekM25) [inline scripts/deployment/archive-packages.sh:1]: ?

**Review by MarekM25 — COMMENTED** [review]
Please consider changing PR title
Please update this, we should mention in release notes that we added new debug_ methods

File: scripts/deployment/archive-packages.sh
Comment (Demuirgos) [reply scripts/deployment/archive-packages.sh:1]: (happened after merging master) I actually dont know what is wrong here, I diffed master's version and these and they are identical
